src/request.jl

```diff
@@ -1,5 +1,5 @@
 function worker_bits()
-  wts = nextpow2(num_threads())
+  wts = nextpow2(Sys.CPU_THREADS) # TODO this is overprovisioning - make sure we are not needlessly iterating through this if we actually have fewer threads available
```

Comment:
I think overprovisioning is fine.
We should consider using `CPUSummary.sys_threads()` here instead.

Also, we should probably be using `worker_bits()` instead of assuming 64, like parts of the code do...